src/markdown-pages/install-with-kurl/cis-compliance.md

```diff
@@ -0,0 +1,50 @@
+---
+path: "/docs/install-with-kurl/cis-compliance"
+date: "2021-12-14"
+weight: 24
+linktitle: "CIS Compliance"
+title: "CIS Compliance"
+isAlpha: false
+---
+The kURL installer can be configured to be Center for Internet Security (CIS) compliant. This is an opt-in feature that is configured in the kURL specification by setting the `kurl.cisCompliance` field to `true`.
+
+To meet CIS compliance, admin.conf permissions are changed from the default `root:sudo 440` to `root:root 444`.
+
+For information about known limitations, see [Known Limitations](#known-limitations).
+
+For more information about CIS security compliance for Kubernetes, see the [CIS benchmark information](https://www.cisecurity.org/benchmark/kubernetes).
+
+This YAMl file example shows a valid specification for CIS compliance:
+
+```
+apiVersion: cluster.kurl.sh/v1beta1
+kind: Installer
+metadata:
+  name: cisCompliance
+spec:
+  kurl:
+    cisCompliance: true
+  kubernetes:
+    version: 1.19.15
+  containerd:
+    version: 1.4.6
+  antrea:
+    version: 1.4.0
+  rook:
+    version: 1.5.12
+  kotsadm:
+    version: 1.58.1
+  ekco:
+    version: 0.13.0
+  registry:
+    version: 2.7.1
+  velero:
+    version: 1.7.1
```

Comment:
How about using the latest spec here?

```suggestion
  kubernetes: 
    version: "1.23.x"
  weave: 
    version: "2.6.x"
  contour: 
    version: "1.20.x"
  prometheus: 
    version: "0.53.x"
  registry: 
    version: "2.7.x"
  containerd: 
    version: "1.4.x"
  ekco: 
    version: "latest"
  minio: 
    version: "2020-01-25T02-50-51Z"
  longhorn: 
    version: "1.2.x"
```

Reply:
latest is deprecated and not used anymore.

Reply:
@pavansokkenagaraj can you suggest a different spec?

Reply:
This spec still needs to change. I don't think we should encourage people to use antrea and rook.

Reply:
Could we just take the current one from kurl.sh and add kots to it? 

apiVersion: "cluster.kurl.sh/v1beta1"
kind: "Installer"
metadata: 
  name: "latest"
spec: 
  kurl:
    cisCompliance: true
  kotsadm:
    version: "latest"
  kubernetes: 
    version: "1.23.x"
  weave: 
    version: "2.6.x"
  contour: 
    version: "1.20.x"
  prometheus: 
    version: "0.53.x"
  registry: 
    version: "2.7.x"
  containerd: 
    version: "1.4.x"
  ekco: 
    version: "latest"
  minio: 
    version: "2020-01-25T02-50-51Z"
  longhorn: 
    version: "1.2.x"